### ISSUE TITLE
Train - Eval Settings

### ISSUE THREAD

**Issue by russellyq** [hailanyi/TED#11]
Hi thanks for your code!

I have questions about your 80%-20% train-eval split. Can you show your txt file for this settings ?

Thanks
Best Regards

**Comment by russellyq** [comment]
Hi @hailanyi ,

I also train the model on 100% training with your pretrained model. However, the results seem not good.

**Comment by hailanyi** [comment]
1. Currently, the data split is not available publicly. Please do not directly partition the data randomly as some frames are from the same sequence. You should ensure the frames of train and val set are from different sequence. On the KITTI data download page, a mapping file, that maps the detection frames to the corresponding sequence id of raw data, is available in the [Download object development kit](https://s3.eu-central-1.amazonaws.com/avg-kitti/devkit_object.zip).  The mapping file can be used to partition your data more reasonably.

2. The best score thresholds of val set and test set are different. The KITTI metric is very sensitive to the object recall. Currently, the default score threshold is relatively high that filters more true positives, resulting in lower recall and AP. Please use lower  score threshold to retain more object candidates, and ensure a higher recall and AP.
